app/controllers/reset_passwords_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ResetPasswordsController < ApplicationController
+
+  def new
```

Comment:
You don't have to have empty controller actions, they are implicit. If you are going to write them, the recommended style is:

```ruby
def new; end
```